docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: "3.7"
```

Comment:
Since this is just the wazimap package, I think this yml file should be as generic as possible and not specific to an instance

Reply:
Sure this was more so that I could run the wazimap package in docker to see any errors that was potentially picked up during startup.